flocx_market/matcher/match_engine.py

```diff
@@ -0,0 +1,33 @@
+from flocx_market.matcher import matcher
+from flocx_market.objects import bid
+from flocx_market.objects import contract
+import datetime
+
+
+def prepare_contract(offers_used, b, context):
+    contract_data = dict(start_time=b.start_time,
+                         time_created=datetime.datetime.utcnow(),
+                         end_time=b.end_time,
+                         cost=b.cost,
+                         project_id=b.project_id,
+                         status='available',
+                         bid_id=b.marketplace_bid_id,
+                         offers=[x.marketplace_offer_id for x in offers_used]
+                         )
+    b.status = 'busy'
```

Comment:
I don't think we want to set the status to `busy`, right? Or if we do, we'd just set it on line 13?

Reply:
Oh, nevermind; b is a bid! Can I suggest using more descriptive parameter names?

Reply:
I renamed it as bid_ as if I name it as the bid that would conflict with BO.